test/beast/core/test_handler.hpp

```diff
@@ -150,6 +151,7 @@ struct legacy_handler
             asio_handler_is_continuation(&h);
             BEAST_EXPECT(hook_invoked);
         }
+#endif // !defined(BOOST_ASIO_NO_DEPRECATED)
```

Comment:
I don't put the trailing comment on ifdefs. It is too much hassle to repeat the expression, which is sometimes long and cumbersome. But that's just my preference.

Reply:
ok